### DISCUSSION TITLE
"Unknown" time sorting behavior

### DISCUSSION THREAD

**Discussion by bef-max** [photoprism/photoprism#4682] [Q&A] [answered]
I have a few thousand pictures from an apple phone that was set to convert to JPEG on import via cable. The metadata was all lost during the conversion. The time taken is therefore all marked as ![Capture](https://github.com/user-attachments/assets/3ae6b87c-82d5-49bf-a238-f7ad7779bbf6). PhotoPrism decides to sort all of these pictures to the top when sorting from newest to oldest even though they are something like eight years old. How can I fix this? Going through then all manually and setting a date is not an option as there are tens of thousands of them.

**Comment by omerdduran** [comment]
Hello! iOS devices often remove metadata when exporting photos. This metadata can include details such as location, date, and camera settings. If you want to keep the metadata, there should be an option that allows you to set it.

**Reply by bef-max** [reply]
Yes, but it's too late for that. I haven't had an iphone since then so I just need to deal with the existing pictures. Is there some kind of batch metadata editor that I could use to keep these pictures from taking over the newest first sort?

**Reply by omerdduran** [reply, marked as answer]
You can easily change the metadata of photos using Exiftool. Check out our documentation for more [info](https://docs.photoprism.app/developer-guide/metadata/exif/edit/#set-createdate). And if you need to change the metadata of more than one photo, you can do that too. Just collect your iPhone photos in a folder, and then use this command to set the date of all your JPEG files to the date you want. 


```bash
exiftool -CreateDate="YYYY:MM:DD HH:MM:SS+02:00" *.jpeg
```
Replace YYYY:MM:DD HH:MM:SS +00:00(time zone)  with your desired date/time. This will update all .jpeg files in the current folder.

**Reply by graciousgrey** [reply]
In case you set the exact same date and time for a batch of photos with location information you might want to disable stacking by place and time --> https://docs.photoprism.app/user-guide/settings/library/#stacks

**Comment by bef-max** [comment]
Is it enabled by default? They aren't stacking right now.

**Reply by omerdduran** [reply]
Yes. Enabled by default

**Comment by bef-max** [comment]
And do they automatically stack if they all say unknown date? I still haven't done anything about this because the proposed solution doesn't work for me as it involves copying around 95 000 media files and collapsing my folder structure. I only marked it as the solution to let the devs get back to work.

**Reply by omerdduran** [reply]
No, photos with unknown dates cannot be automatically stacked. If you add -overwrite_original to the end of the command, it only changes the metadata of the photos without any copying. However, it may not make sense to set all 95,000 photos to be taken on the same date and at the same time.

**Comment by bef-max** [comment]
Is there a way to do this without collapsing folder structure?
